hws/mlx_hw_steering_parser.py

```diff
@@ -118,15 +120,15 @@ def dr_parse_csv_file(csv_file, load_to_db):
         elif line[0] == MLX5DR_DEBUG_RES_TYPE_CONTEXT_STC:
             obj.load_to_db()
         elif line[0] == MLX5DR_DEBUG_RES_TYPE_HW_RRESOURCES_DUMP_START:
-            if not(load_to_db):
+            if not (load_to_db):
                 return ctx
             _config_args["hw_resources_present"] = True
         elif line[0] == MLX5DR_DEBUG_RES_TYPE_FW_STE:
             if last_fw_ste != None:
                 last_fw_ste.add_stes_range(min_ste_addr, max_ste_addr)
             obj.init_fw_ste_db()
-            max_ste_addr = '0x00000000'
-            min_ste_addr = '0xffffffff'
+            max_ste_addr = "0x00000000"
+            min_ste_addr = "0xffffffff"
```

Comment:
PEP 8 doesn't specify anything about this. Other places say "Use single quotes for strings, unless you need to use a quote character inside the string, in which case you can use double quotes".
Anyway, no need for all these quote changes